testnet/stacks-node/src/neon_node.rs

```diff
@@ -794,7 +778,7 @@ fn spawn_miner_relayer(
                                 && burn_hash == mined_burn_hash
                             {
                                 // we won!
-                                info!("Won sortition!";
+                                debug!("Won sortition!";
```

Comment:
I'd suggest making this an `info!` log, everyone likes to know when they win!

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2234/commits/445e25c5863951ce0606d59b4cf66b08529bf801